docs/user-guide/projects/project-lists/README.md

```diff
@@ -40,37 +40,61 @@ You can use the Project overview page to **create a multi-project status dashboa
 
 ![Open a project description in the project lists view in OpenProject](expand-link-project-description.png)
 
-With the horizontal **three dots** icon on the right side of the list you can open **further features**, such as [creating a new subproject](../#create-a-subproject), [project settings](../project-settings), [archiving a project](../#archive-a-project), [copying](../#copy-a-project) and [deleting a project](../#delete-a-project). Please note that you have to be a System Administrator in OpenProject to access these features. Find out how to un-archive projects [here](../#archive-a-project).
+With the horizontal **three dots** icon on the right side of the list you can open **further features**, such as [creating a new subproject](../#create-a-subproject), [project settings](../project-settings), [add a project to favorites](../../project-overview/#mark-a-project-as-favorite), [archiving a project](../#archive-a-project), [copying](../#copy-a-project) and [deleting a project](../#delete-a-project). Please note that you have to be a System Administrator in OpenProject to access these features. Find out how to un-archive projects [here](../#archive-a-project).
 
 ![new subproject project list](new-subproject-project-list.png)
 
+
+
+## Configure project lists view
+
 You can choose the **columns displayed by default** in the [Project lists settings](../../../system-admin-guide/projects/project-lists) in the Administration.
 
 You can add the columns, as well as define the order of the columns by using the **Configure view** modal. Navigate to it via the menu in the far right corner (three dots) and click **Configure view**.
 
 ![Configure view of project lists in OpenProject](configure-view-project-list.png)
 
-A dialogue will open, allowing you to manage and reorder columns. Click **Apply** to see the changes.
+A dialogue will open, allowing you to manage and reorder columns under the tab *Columns*.
 
 ![Configuration form for project lists in OpenProject](configure-view-form-project-list.png)
 
 To change the order of the displayed [project attributes](../../../system-admin-guide/projects/project-lists) (columns) follow the instructions here: [Displaying a project attribute (formerly called custom field)](../../../system-admin-guide/projects/project-lists).
 
+Under the tab *Sort* you can select the sort order for the project lists. You can select up to three criteria and define the sorting order (ascending or descending). You will be able to change the sorting order later by clicking the column header.
+
+![Define sort order for project lists in OpenProject](configure-view-form-project-list-sort-order.png)
+
+ Click **Apply** to see the changes.
+
+You will now be able to save your adjust list by clicking the *Save as* link. Alternatively you can click the *More* icon and select the *Save as* option from the dropdown menu that will open.
```

Comment:
I fear this sentence is not 100% correct. The options displayed as link depends if you are in a static list or in a private one. As static lists can not be modified the **Save as** option appears, but if you are in your own private list then the link will be **Save**.